docs/conf.py

```diff
@@ -21,47 +21,155 @@
 copyright = "%s, %s" % (datetime.date.today().year, author)
 release = snapcraft.__version__
 
+# Open Graph configuration - defines what is displayed in the website preview
+ogp_site_url = "https://canonical-snapcraft.readthedocs-hosted.com"
+ogp_site_name = project
+ogp_image = "https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg"
+
+# Update with the favicon for your product
+html_favicon = "sphinx-resources/.sphinx/_static/favicon.png"
+
+html_context = {
+    # Change to the discourse instance you want to be able to link to
+    # (use an empty value if you don't want to link)
+    "discourse": "https://forum.snapcraft.io",
+    # Change to the GitHub info for your project
+    "github_url": "https://github.com/canonical/snapcraft",
+    # Change to the branch for this version of the documentation
+    "github_version": "main",
+    # Change to the folder that contains the documentation (usually "/" or "/docs/")
+    "github_folder": "/docs/",
+    # Change to an empty value if your GitHub repo doesn't have issues enabled
+    "github_issues": "enabled",
```

Comment:
```suggestion
html_context = {
    "discourse": "https://forum.snapcraft.io",
    "github_url": "https://github.com/canonical/snapcraft",
    "github_version": "main",
    "github_folder": "/docs/",
    "github_issues": "enabled",
```

Just taking the comments that were intended for the starter pack out :-)